src/UNL/MediaHub/FeedList.php

```diff
@@ -51,7 +51,7 @@ public function filterInputOptions()
      * 
      * @see UNL/MediaHub/UNL_MediaHub_List#setOrderBy()
      */
-    function setOrderBy(Doctrine_Query &$query)
+    function setOrderBy(Doctrine_Query_Abstract &$query)
```

Comment:
Please remove the `&` style of param passing (php4). Objects should always come by reference.